src/commands/install.rs

```diff
@@ -35,6 +37,12 @@ pub fn install() -> Result<()> {
         Err(_) => bail!("Failed to fetch latest fuel-core release tag from GitHub API"),
     };
 
+    let fuelup_path = home_dir().unwrap().join(Path::new(FUELUP_BIN_PATH));
```

Comment:
Let's re-use the `fuelup_bin_dir` function here, and maybe change the `fuelup_path` binding to reflect it.